hooks/amqp-relation-joined.ps1

```diff
@@ -19,12 +20,13 @@ try {
 
     $rids = Get-JujuRelationIds -Relation "amqp"
     foreach ($rid in $rids){
-        $ret = Set-JujuRelation -RelationId $rid -Settings $relation_set
-        if ($ret -eq $false){
+        try {

```

Comment:
I think this should error out the hook. So just leave the all to Set-JujuRelation